master/reference/advanced/etcd-rbac/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Setting up etcd certificates for RBAC
+---
+
+When using etcd it is a good idea to protect the data stored there.  This is
+even more true when you have multiple components using a common etcd cluster.
+
+## Why you might be interested in this guide
+
+- You want to restrict the operations that support staff are able to perform
+  to minimize accidental corruption or deletion of etcd data.
+- You are using the etcd cluster with multiple orchestrators (Kubernetes,
+  docker, Calico) or multiple tenants and want to prevent access between the
+  multiple tenants.
+- You want to restrict the read/write access of the various Calico components
+  as an additional safety measure.
```

Comment:
I thought that was one of the main motivations for this doc.  If creating separate component roles is not necessary then I agree it contributes to complexity, the whole calico-etcdv2-paths document itself might not need to exist at all.
@robbrockbank Did I misunderstand things and the goal was to only separate Calico from other non-Calico etcd data or were we interested in actually separating the individual Calico components? That would include being able to restrict the modifications that users could make with calicoctl, but still if the purpose was being able to restrict users of calicoctl and not the individual calico components that doc could be reduced.